packages/astro/src/core/logger.ts

```diff
@@ -89,6 +89,12 @@ export const levels: Record<LoggerLevel, number> = {
 	silent: 90,
 };
 
+export function enableVerboseLogging() {
+	debugPackage.enable('*,-babel'); 
+	debug('cli', '--verbose flag enabled! Enabling DEBUG="*,-babel"');
+	debug('cli', 'Set the DEBUG env variable directly for more control. Example: "DEBUG=astro:*,vite:* astro build".');
```

Comment:
Wouldn't the first line override the the env var, making useless the advice below? [Reference](https://github.com/debug-js/debug#set-dynamically)

Reply:
So I'm trying to communicate: "hey, use `DEBUG=` _instead_ of --verbose if you need more control over what to log." Was that how you read it? If not, is there a way to rewrite this to be more clear? Definitely open to suggestions!

Reply:
I understood I'd had to use DEBUG along verbose. Maybe, you could check if DEBUG is already set